src/rustdoc-json-types/lib.rs

```diff
@@ -30,7 +30,7 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 /// This integer is incremented with every breaking change to the API,
 /// and is returned along with the JSON blob as [`Crate::format_version`].
 /// Consuming code should assert that this value matches the format version(s) that it supports.
-pub const FORMAT_VERSION: u32 = 40;
+pub const FORMAT_VERSION: u32 = 41;
```

Comment:
This is taking place at the same time as #138109, which also (currently) makes the 40->41 change. Once one of these merges, the other should rebase then bump again. See #94591 for details.